app/models/ui/Sequence.scala

```diff
@@ -16,18 +16,10 @@
 
 package models.ui
 
-case class NonExclusiveSequence(text: Text,
-                                hint: Option[Text],
-                                options: Seq[Text],
-                                body: Seq[UIComponent],
-                                errorMsgs: Seq[ErrorMsg])
-  extends NonExclusiveSequenceFormComponent
+case class SequenceAnswer(text: Text, hint: Option[Text], exclusive: Boolean = false) extends UIComponent
```

Comment:
In a set of answers, there can only ever be one excusive answer, would be useful to have the Sequence/Answer types reflect ( the boolean flag would allow multiple exclusive answers)

Reply:
This can be changed, but it is an additional refactor for tests and such down the line

Reply:
it would improve the code, I can help with test changes